src/Week1/Homework1.hs

```diff
@@ -9,21 +9,22 @@ toDigits num =
   helper n l =
    if n < 1
    then l
-   else helper (div n 10) ((mod n 10):l)
+   else helper a $ b:l
+    where
+      (a,b) = divMod n 10
```

Comment:
I think, what you can do here, is to change order of arguments in `helper` function from `Integer -> [Integer]` to `[Integer] -> Integer`. Then you can eta-reduce and write just:

```haskell
toDigits = helper []
```

Regarding `divMod`: your current solution is okay. I don't see any nicer way to do this.